cogs/translate.py

```diff
@@ -45,13 +45,18 @@ async def translate(self, ctx: Context, source: str, target: str, text: str):
                     translatedText = " ".join(json["translatedText"])
                     await ctx.send(f'"{translatedText}"')
         except Exception as e:
-            logging.error(f"Failed")
+            logging.error(f"Failed to translate: {e}")
             return
 
     @translate.error
     async def translate_handler(self, ctx, error):
+        message = ":x: "
+        if isinstance(error, commands.MissingRequiredArgument):
+            message += 'Missing required arguments!\nNeeds to follow this format: `!translate {source} {target} "text to be translated"`\nReplace `source` and `target` with their code found in `!languages`.'
+        else:
+            message += f"Command failed to execute due to: ```\n{error}\n```"
```

Comment:
i swear to god this needs to be in a decorator

Reply:
that's ur specialty